services/bstats/bstats.tester.js

```diff
@@ -0,0 +1,33 @@
+'use strict'
+
+const Joi = require('joi')
+
+const ServiceTester = require('../service-tester')
+const t = (module.exports = new ServiceTester({
+  id: 'bstats',
+  title: 'bStats badges',
+}))
+
+const withRegex = re => Joi.string().regex(re)
+
```

Comment:
For future reference, you'll find a lot of these sorts of validators defined in the test-validators file (`services/test-validators.js`) so you'll generally want to check there first to see of something already exists (`withRegex` already does for example)